Classes/AMPPreviewController.m

```diff
@@ -79,12 +83,12 @@ - (id)initWithRemoteFile:(NSURL *)remoteUrl title:(NSString *)title {
     return self;
 }
 
-- (void)viewDidAppear:(BOOL)animated {
```

Comment:
Could you explain why this change is necessary? 
To be honest I don't remember why I used `viewDidAppear` so many years ago, but I would like to know if the change is required to make it work nowdays.

Reply:
The issue with this is that it shows a blank screen for a second before showing the actual content (in case the remote file was already cached) which is unnecessary.

By moving it to `viewDidLoad` we get presented with the actual content right away.

Reply:
Yes make sense. I'm pretty sure it caused issues in old OSs that's why I moved the code here. But if it's working now I agree `viewDidLoad` is a better place.